src/Settings.js

```diff
@@ -23,6 +23,7 @@ function Settings(options) {
     this.displayMode = get(options.displayMode, false);
     this.throwOnError = get(options.throwOnError, true);
     this.errorColor = get(options.errorColor, "#cc0000");
+    this.macros = options.macros;
```

Comment:
The `MacroExpander` constructor already provides a default. The `Parser` constructor passes `settings && settings.macros`. I guess that the situation where `settings` is undefined shouldn't really be possible with out public API. So I could just pass on `settings.macros` unconditionally, and avoid the default in `MacroExpander` as well. Yes, this should work nicely.